### DISCUSSION TITLE
Start the recording via an external trigger

### DISCUSSION THREAD

**Discussion by digi303** [blakeblackshear/frigate#5322] [Ask A Question]
I sometimes have the problem that movements are not detected at night - but I would like to trigger a recording / object detection when opening the utility room door. Since the opening is detected via a contact sensor, I would definitely have a trigger already....

Can this already be realised?
Or is this more of a feature request?

Best regards

digi

**Comment by NickM-27** [comment]
Hi, for one it is recommended to set `motion -> improve_contrast` to improve detection. This can be done via the config file or via the home assistant integration (for example to set it only at night and disable during the day).

There is already a feature request for this #1063

**Reply by digi303** [reply]
okay, ive "found" the discussion. Thanks for the hint "motion -> improve_contrast" i will try this one at night.
